File: App.Admin/src/api/services/Admin/organization/models/QueryOrganizationDetailOutput.ts
/**
 * Fast.Admin.Service.Organization.Dto.QueryOrganizationDetailOutput 获取机构详情输出
 */
export interface QueryOrganizationDetailOutput {
  /**
   * 机构Id
   */
  orgId?: number;
  /**
   * 父级Id
   */
  parentId?: number;
  /**
   * 父级名称
   */
  parentName?: string;
  /**
   * 父级Id集合
   */
  parentIds?: Array<number>;
  /**
   * 父级名称集合
   */
  parentNames?: Array<string>;
  /**
   * 机构名称
   */
  orgName?: string;
  /**
   * 机构编码
   */
  orgCode?: string;
  /**
   * 联系人
   */
  contacts?: string;
  /**
   * 电话
   */
  phone?: string;
  /**
   * 邮箱
   */
  email?: string;
  /**
   * 排序
   */
  sort?: number;
  /**
   * 备注
   */
  remark?: string;
  /**
   * 
   */
  departmentName?: string;
  /**
   * 
   */
  createdUserName?: string;
  /**
   * 
   */
  createdTime?: Date;
  /**
   * 
   */
  updatedUserName?: string;
  /**
   * 
   */
  updatedTime?: Date;
  /**
   * 
   */
  rowVersion?: number;
}

File: Api.Server/src/Admin.Service/Organization/Dto/QueryOrganizationDetailOutput.cs
// Copyright © 2018-Present 小方
// SPDX-License-Identifier: Apache-2.0
// 
// 本文件依据 Apache License 2.0 授权，完整条款见仓库根目录 LICENSE。
// 本软件按“原样”提供，相关免责声明及责任限制以许可证及适用法律为准。
// 版权来源、合法使用与二次开发责任说明见仓库根目录 README.md。

namespace Fast.Admin.Service.Organization.Dto;

/// <summary>
/// 获取机构详情输出
/// </summary>
public class QueryOrganizationDetailOutput : PagedOutput
{
    /// <summary>
    /// 机构Id
    /// </summary>
    public long OrgId { get; set; }

    /// <summary>
    /// 父级Id
    /// </summary>
    public long ParentId { get; set; }

    /// <summary>
    /// 父级名称
    /// </summary>
    public string ParentName { get; set; }

    /// <summary>
    /// 父级Id集合
    /// </summary>
    [SugarColumn(IsJson = true)]
    public List<long> ParentIds { get; set; }

    /// <summary>
    /// 父级名称集合
    /// </summary>
    [SugarColumn(IsJson = true)]
    public List<string> ParentNames { get; set; }

    /// <summary>
    /// 机构名称
    /// </summary>
    public string OrgName { get; set; }

    /// <summary>
    /// 机构编码
    /// </summary>
    public string OrgCode { get; set; }

    /// <summary>
    /// 联系人
    /// </summary>
    public string Contacts { get; set; }

    /// <summary>
    /// 电话
    /// </summary>
    public string Phone { get; set; }

    /// <summary>
    /// 邮箱
    /// </summary>
    public string Email { get; set; }

    /// <summary>
    /// 排序
    /// </summary>
    public int Sort { get; set; }

    /// <summary>
    /// 数据公开
    /// </summary>
    public bool DataPublic { get; set; }

    /// <summary>
    /// 备注
    /// </summary>
    public string Remark { get; set; }
}
